### PR TITLE
Update gitignore

### PR DESCRIPTION
**Pull request by halink0803**
+ remove cmd/verification/verification
+ remove cmd/comparates/comparates
+ remove cmd/ropsten.db

### CODE REVIEW

**Comment by halink0803** [comment]
[Binary files are commited to git repository](https://trello.com/c/WlgiZIgp/39-binary-files-are-commited-to-git-repository)

**Comment by halink0803** [comment]
[Binary files are commited to git repository](https://trello.com/c/WlgiZIgp/39-binary-files-are-commited-to-git-repository)

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,6 @@ debugger.txt
 log.log
 run.sh
 test/
+cmd/verification/verification
+cmd/comparerates/comparerates
+cmd/ropsten.db
```
Comment (favadi) [inline .gitignore]: @tranvictor @cyberliem could you confirm that this file should be removed from repository?

Comment (cyberliem) [reply .gitignore]: I second the motion. But why not cmd/*.db?

Comment (halink0803) [reply .gitignore]: @cyberliem this require programmer to be careful when adding db file. I don't think *.db or *.xxx is a good options.

Comment (favadi) [reply .gitignore]: Actually all cmd/*.db is ignored already: https://github.com/KyberNetwork/reserve-data/blob/b2086adc796ef7ed31daa668bbb4d2936b7726b0/.gitignore#L32.

Comment (favadi) [reply .gitignore]: this line should be removed.

Comment (tranvictor) [reply .gitignore]: yes, good to go

Comment (cyberliem) [reply .gitignore]: @halink0803 Have you removed this yet? Seems like you havent.

Comment (halink0803) [reply .gitignore]: @favadi @cyberliem Just remove it, please review.

**Pull request merged**